### DISCUSSION TITLE
Able to set endDate less than startDate for the content types

### DISCUSSION THREAD

**Discussion by vinod77** [strapi/strapi#7088] [Legacy Topics]
**Describe the bug**
Able to set endDate less than startDate for the content types

**Steps to reproduce the behavior**

1. Go to: any of the Content-Types which has both `startDate` & `endDate` fields
2. Select `startDate` 
3. And select `endDate` less than `startDate`
4. Observe: We are able to set `endDate` less than `startDate` - Which is wrong

**Expected behavior**
`endDate` should always be greater than `startDate`

**Screenshots**

<img width="1650" alt="Strapi - Content Manager 2020-07-20 12-41-05" src="https://user-images.githubusercontent.com/23507845/87968703-6fe61700-ca86-11ea-9d89-96225e145f60.png">

**System**

Node version: `v10.15.3`
NPM version: `6.4.1`
Strapi version: `3.0.0-beta.20.3`
Database: `Mongo` & `S3` File storage for File uploads
Operating system: `MAC`

**Comment by derrickmehaffy** [comment]
I see in your screenshot you are using v3.0.2 but you listed `3.0.0-beta.20.3` as your version, can you please confirm which version of Strapi you are using?

Also currently there is not a direct way to base one field off of another (other than with the UID field which is attached to another to generate the UID automatically).

**Comment by vinod77** [comment]
Hi @derrickmehaffy 

I am using 3.0.0-beta.20.3. I sent you screenshot from default strapi version: `v3.0.2`
But I see this issue exists on all the Strapi version (including latest)

So how can we validate it on the Front-End itself? We are just adding similar Date fields for startDate & endDate.
How does Strapi UI knows what is startDate & what is endDate ?

Please let us know if there is any solution to fix this on Strapi Front-end itself (if possible by overriding Strapi code)

**Comment by derrickmehaffy** [comment]
This would likely be something you would add to the model lifecycles (check to make sure `startDate isn't greater than endDate` but in the beta version you are using those model lifecycles are broken.

Please due note that Strapi beta versions are no longer supported, so if you only see an issue in the Beta the issue won't be fixed.

**Comment by vinod77** [comment]
HI @derrickmehaffy 

Looks your solution is for the back-end. 
But we want to do the validation on the front-end itself. If we can do validation on front-end, we can show the error message to the User once wrong date is selected.

How do we do it on the front-end?

Thanks!

**Comment by derrickmehaffy** [comment]
That I don't have an answer for, though I am going to move this over to a discussion since we know this isn't a bug.
